presto-docs/src/main/sphinx/functions/noisy.rst

```diff
@@ -0,0 +1,200 @@
+=========================
+Noisy Aggregate Functions
+=========================
+
+Noisy aggregate functions are aggregate functions that provide random, noisy
+approximations of common aggregations like :func:`sum`, :func:`count`, and
+:func:`approx_distinct` as well as sketches like :func:`approx_set`. By
+injecting random noise into results, noisy aggregation functions make it more
+difficult to determine or confirm the exact data that was aggregated.
+
+.. note::
+
+    While many of these functions resemble
+    `differential privacy <https://en.wikipedia.org/wiki/Differential_privacy>`_
+    mechanisms, it is important to note that neither the values returned by
+    these functions nor the query results that incorporate these functions
+    are differentially private in general. See `Limitations`_ below for more
+    details. Users who wish to support a strong privacy guarantee should
+    discuss with a suitable technical expert first.
+
+Counts, Sums, and Averages
+--------------------------
+
+.. function:: noisy_count_gaussian(x, noise_scale[, random_seed]) -> bigint
+
+    Counts the non-``NULL`` values in ``x`` and then adds random Gaussian noise with 0
```

Comment:
What's x? (a column I assume)

Reply:
Indeed, I've renamed all these from `x` to `col`, which is hopefully more clear.